bazel/cas/service.go

```diff
@@ -425,6 +439,10 @@ func (s *casServer) GetActionResult(ctx context.Context,
 	if !s.IsInitialized() {
 		return nil, status.Error(codes.Internal, "Server not initialized")
 	}
+	if s.concurrent != nil {
```

Comment:
Not sure why you'd limit action results. I thought those are supposed to be small but I could be mistaken?

Reply:
Fair enough, if you have other plans in mind.